syntax = "proto3";

service Aggregation {
    rpc SendDataPath(simpleStringRequest) returns (simpleStringResponse);
    rpc SendParamsFTP(infoFTP_Request) returns (simpleStringResponse);
    rpc MakeAggregation(Agregregation) returns (Agregregation);
    rpc sendGraphData(simpleStringRequest) returns (simpleStringRequest);
}

message simpleStringRequest{
    string req = 1;
}

message simpleStringResponse{
    string resposta = 1;
}

message infoFTP_Request{
    string host = 1;
    string porta = 2;
    string usuario = 3;
    string senha = 4;
}


message Agregregation{
    string data_inicio  = 1;
    string data_fim =     2;
    string vdc  =         3;
    string idc =          4;
    string vac =          5;
    string iac =          6;
    string freq =         7;
    string pac =          8;
    string ene =          9;
    string whs =          10;
}

